schema.graphql

```diff
@@ -322,12 +322,18 @@ type LBPair @entity {
 type Bin @entity {
   id: ID! # lbPair address + # + binId
   lbPair: LBPair!
+
   priceX: BigDecimal!
   priceY: BigDecimal!
-  totalSupply: BigInt! 
+
+  totalSupply: BigInt!
   reserveX: BigDecimal!
   reserveY: BigDecimal!
+
   binId: BigInt!
+
+  liquidityProviders: [String!]!
```

Comment:
will change this to `liquidityProviders: [UserBinLiquidity!]! @derivedFrom(field: "lbPairBinId")` so that we don't have to add/remove userIds manually.

Reply:
ah this won't work because derived array of child entities does not get populated until the query happens. But we need this during indexing. so reverting back to `liquidityProviders: [String!]!`